README.md

```diff
@@ -50,18 +50,18 @@ julia> using TestImages, ExifViewer
 julia> filepath = testimage("earth_apollo17.jpg", download_only=true)
 julia> io = open(filepath, "r")
 julia> read_tags(io; read_all=false, tags=["EXIF_TAG_FLASH_PIX_VERSION", "EXIF_TAG_ORIENTATION"])
-Dict{Any, Any} with 2 entries:
+Dict{String, String} with 2 entries:
```

Comment:
I should have fixed these in previous PRs where I switched from `Dict{Any, Any}` to `Dict{String, String}` for performance. Thanks!